src/irisnet/noncore/cutestuff/httpconnect.cpp

```diff
@@ -160,7 +160,7 @@ void HttpConnect::connectToHost(const QString &proxyHost, int proxyPort, const Q
     else
         fprintf(stderr, ", auth {%s,%s}\n", qPrintable(d->user), qPrintable(d->pass));
 #endif
-    d->sock.connectToHost(d->host, d->port);
+    d->sock.connectToHost(d->host, quintptr(d->port));
```

Comment:
better change HttpConnect::connectToHost function signature to use quint16 for the port param.
Maybe some other classes also suffer from this problem. I'd grep all connectTohost